bciers/apps/reporting/src/app/(authenticated)/authenticated/facilities/page.tsx

```diff
@@ -0,0 +1,45 @@
+"use client"
+
+import React from 'react';
+import { actionHandler } from "@/app/utils/actions";
+
+const Page: React.FC = () => {
+  const fetchOperators = async () => {
+    try {
+      const operatorList = await actionHandler(
+        "registration/operations",
+        "GET",
+      );
+      return operatorList.data;
+    } catch (error) {
+      // Handle the error here or rethrow it to handle it at a higher level
+      throw error;
+    }
+  };
+
+  return (
+    <div>
+      <h1>Facilities and operators page</h1>
+      <h2>Operators:</h2>
+      <OperatorsDisplay fetchOperators={fetchOperators} />
+    </div>
+  );
+};
+
+const OperatorsDisplay: React.FC<{ fetchOperators: () => Promise<any> }> = ({ fetchOperators }) => {
```

Comment:
- no reason to have the function as a parameter here, it's not required to be in the scope of the component
- make the component an async function

Something like:
```typescript
const OperatorsComponent = async () => { ... }
```